Cookbook/Technical-Documents/OutOffOfficeRequest.md

```diff
@@ -0,0 +1,97 @@
+
+There are different types of OOO and each one of them have different requirements for requesting and for being notified.
+
+## Holidays 🌴
+
+To promote a healthy work-life balance we should take 25 holiday days per year. You can book your holidays at your convience, you might just need to sync first with your Squad, including the iOS engineers in it, to see if every one is OK with it. 
+
+Ideally, you should give at least 2 weeks notice before taking any holiday but there are exceptions for emergency situations. If you want to book more than 5 consecutive days you ideally should give 4 weeks notice.
+
+### Request
+
+1. Check with the iOS Engineers in your squad that the work is covered during you absence.
+1. Ask for approval from your PM - only if approved you can proceed to the next steps.
+1. Communicate it to your line manager and mention your PM approval
+1. Request in Bamboo
+
+### Notify
+
+1. [Send an event](#steps-to-configure-the-outlook-event) to the iOS Team Calendar and to your Squad email group
+1. Update the [Team Plan document](https://docs.google.com/spreadsheets/d/1kdY3edy_TeqIGH_7VnZzElxgVo_qD2z4EF-arWNShyw/edit?usp=sharing) 
+1. Notify the iOS Team and your Squad via Slack on the working day before you go on holidays.
+1. [Enable automatic reply message](#steps-to-configure-automatic-reply) in your Babylon Outlook account.
+1. At the end of your last working day before going on holidays change your Slack status to 🌴**On holiday** and set to clear after your last holiday day. 
+
+If there is a big impact in the iOS Team or in your Squad due to you being away, please communicate a few days before in the team specific slack channel and do a handover if needed.
+
+#### Steps to Configure the Outlook event
+1. Open Outlook 
+1. Select Calendar
+1. Select Meeting
+1. To: `<iOS calendar email group>`, `<your squad email group>`
+1. Subject: `<Your name>` Holidays 🌴
+1. Duration: All-day event
+1. Show as: Free
+1. Reminder: None
+1. Select Save & Close
+
+#### Steps to Configure automatic reply
+1. Open Outlook 
+1. Select Tools
+1. Select Out of Office
+1. ✔️ Send automatic replies for account `<your outlook account>`
+1. Reply once to each sender with: `<Define a message mentioning the period you are on holidays>`
+1. ✔️ Only send replies during this time period
+1. Start time: First day of your holiday 
+1. End time: Last day of your holiday
+
+### Before leaving
+
+When we leave on holidays we should make sure we leave our work either finished or delegated to someone else. There are a couple of steps that should be performed to reduce the impact of the absence.
+
+1. Try to finish any work in progress before you go.
+1. Resolve the reviews on your opened PRs and try to get them merged.
+1. In the event of not being able to complete 1. and/or 2. please sync with another iOS Engineer, ideally someone from your squad, and do a handover. When handing over an opened PR please make sure you add the iOS Engineer as `Assignee`.
```

Comment:
1. Also be sure that your tickets on JIRA are up-to-date, and especially mention any relevant info (like pending state, what's left to be done, who to contact about this particular ticket in your absence...) in the tickets that are not finished yet at the time you leave.